### PR TITLE
Bugfixes + Cut 1.13.1 + documenting release process.

### PR DESCRIPTION
**Pull request by bwplotka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +69,22 @@ See the [contributing guidelines](CONTRIBUTING.md) and the
...
+To cut minor version:
+
+1. Create new branch `release-<major>.<minor>` on top of main commit you want to cut version from and push it.
+2. Create new branch on top of release branch.
+3. Change `VERSION` file.
+4. Update `CHANGELOG` (only user-impacting changes to mention).
+5. Create PR, get it reviewed.
+6. Once merged, create release for `release-<major>.<minor>` tag on GitHub with `<version> / <date>` title.
+7. Announce on prometheus-announce mailing list, slack and Twitter.
+8. Merge release branch back to main using "merge without squashing" approach (!).
```
Comment (bwplotka) [inline README.md:84]: This will create problems if there will be conflicts I believe. Any ideas how to proceed to still not squash? @beorn7

Comment (beorn7) [reply README.md:84]: I usually checkout the release branch in a new branch, e.g. `beorn7/resolve-conflicts`, fix the merge problems there, and then do a PR into main from the new branch. In that way, you still get all the commits in the release branch back into main, but leave the release branch alone.

Comment (bwplotka) [reply README.md:84]: I was looking exactly for this tip ❤️ will add to list

**Comment by bwplotka** [comment]
cc @bboreham

**Review by kakkoyun — APPROVED** [review]
LGTM! Thanks for taking care of it!

**Pull request merged**